src/Pages/ohyeonkwon/Login/Login.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './Login.scss';
+//import { Link } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: '',
+      pw: '',
+      nickname: 'nickname',
+      phone_number: '010-1234-5626',
+      background: '',
+      disabled: true,
+    };
+  }
+
+  goMain = () => {
+    this.props.history.push('/mainH');
+    // fetch('http://10.58.7.108:8000/user/signin', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     email: this.state.id,
+    //     password: this.state.pw,
+    //     nickname: this.state.nickname,
+    //     phone_number: this.state.phone_number,
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => {
+    //     if (result.token) {
+    //       //성공
+    //       alert('success');
+    //       this.props.history.push('/mainH');
+    //     } else {
+    //       //fail
+    //     }
+    //   });
+  };
+
+  inputValue = e => {
+    // input에 작성한 값
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+    console.log(this.state.id, this.state.pw);
+  };
+
+  colorChange = () => {
+    if (this.state.id.includes('@') && this.state.pw.length > 4) {
+      this.setState({
+        background: 'black',
+        disabled: false,
+      });
+    }
+  };
+
+  render() {
+    return (
+      <div>
+        <title>Westagram</title>
+        <body>
+          <div className="logo">
+            <h1>Westagram</h1>
+          </div>
+          <section className="login_section">
```

Comment:
```suggestion
  render() {
    return (
      <div className="login">
          <h1>Westagram</h1>
          <section className="login_section">
```
1. title 태그 및 body 태그는 해당 부분에 들어갈 내용이 아닙니다! 이미 index.html에 있어요!
2. 불필요한 depth를 줄여주세요! h1을 바로 사용하면 되는데, 굳이 div로 감쌀 필요는 없습니다!